circle.yml

```diff
@@ -17,10 +17,9 @@ dependencies:
     - "project/target/streams"
   pre:
     - cd $(dirname "$0");
-    - rm -rf node_modules/ public/jspm_packages/
+    - rm -rf node_modules/
     - npm install
```

Comment:
Worth switching to using [yarn on circleci](https://circleci.com/docs/1.0/yarn/) as we recommend it locally?